web/src/components/navbar/NavBarDashboardLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { useRouter } from "next/router";
+import { ReactNode } from "react";
+
+interface Props {
+  businessName: string;
+  stringToDisplay: string;
+  className?: string;
+}
+
+export const NavBarDashboardLink = (props: Props): ReactNode => {
+  const router = useRouter();
+  return (
+    <div>
+      <UnStyledButton
+        onClick={(): void => {
+          router.push(ROUTES.dashboard);
+        }}
+        ariaLabel={`My Account for ${props.businessName} Home`}
```

Comment:
[sand] `businessName` is allowed to be undefined. It will be until the user supplies a business name, which may be a while. During that time the ariaLabel will be `My Account for  Home`. What if we change this to say either `My Account Home` or `My Account for ABC Business Home`?